src/routers/comments.ts

```diff
@@ -0,0 +1,13 @@
+import type { PostgresJsDatabase } from 'drizzle-orm/postgres-js';
+import { default as express } from 'express';
+import type * as db from '../db';
+import { isLoggedIn } from '../middleware/isLoggedIn';
+import { saveComment } from '../services/comments';
+import { getCommentsForOption } from '../services/comments';
+const router = express.Router();
+
+export function commentsRouter({ dbPool }: { dbPool: PostgresJsDatabase<typeof db> }) {
+  router.get('/:questionOptionId/comment-data', isLoggedIn(), getCommentsForOption(dbPool));
+  router.post('/:questionOptionId/comment', isLoggedIn(), saveComment(dbPool));
+  return router;
+}
```

Comment:
i think this should be part of an options router if there is any, but it should start with options/:optionId/comment. Also change comment-data to comments

Reply:
Deleted the comments router and moved the functionality to the newly created options router (merged develop into this branch to get the option router).